### PR TITLE
feat(section): added helper funtions for manipulating sections

### PR DESCRIPTION
**Pull request by bestinbthomas**

### CODE REVIEW

File: src/helpers/section.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import Cell from '../Interfaces/model/cell.interface';
+import SectionModel from '../Models/section.model';
+
+/**
+ * create a new section with no cells
+ * @param cells cells to add to section
+ */
+export const createSection = async (cells?: Cell[]) => {
+    return await SectionModel.create({ cells: cells ? cells : [] });
```
Comment (Pavi114) [inline src/helpers/section.ts]: A section should have atleast one cell. So create a cell and push it to section?

File: src/helpers/section.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    return await SectionModel.findById(id);
+};
+
+/**
+ * Add cell to section
+ * @param id id of section
+ * @param cell cell to add
+ */
+export const addCellToSection = async (id: string, cell: Cell) => {
+    return await SectionModel.update({ _id: id }, { $push: { cells: cell } });
```
Comment (Pavi114) [inline src/helpers/section.ts]: There should be a max limit on the number of cells a section can have. Add a check for that?

**Review by Pavi114 — CHANGES_REQUESTED** [review]

File: src/helpers/section.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+export const createSection = async (cells?: Cell[]) => {
+    return await SectionModel.create({ cells: cells ? cells : [] });
+};
+
+/**
+ * get the section with _id provided, null if not found
+ * @param id _id of the section
+ */
+export const getSection = async (id: string) => {
+    return await SectionModel.findById(id);
```
Comment (Pavi114) [inline src/helpers/section.ts]: Change type of id to ObjectId

File: src/helpers/section.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ */
+export const createSection = async (cells?: Cell[]) => {
+    return await SectionModel.create({ cells: cells ? cells : [] });
+};
+
+/**
+ * get the section with _id provided, null if not found
+ * @param id _id of the section
+ */
+export const getSection = async (id: string) => {
```
Comment (Jayashrri) [inline src/helpers/section.ts]: Change id type to ObjectId

File: src/helpers/section.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import Cell from '../Interfaces/model/cell.interface';
+import SectionModel from '../Models/section.model';
```
Comment (Jayashrri) [inline src/helpers/section.ts]: Use camelCase please

**Review by Jayashrri — CHANGES_REQUESTED** [review]

File: src/helpers/section.ts (hunk 5 above)
Comment (AananthV) [reply src/helpers/section.ts]: While loading models / external files CapsIsBetter

**Review by Pavi114 — APPROVED** [review]

**Pull request merged**